modules/backend/layouts/default.htm

```diff
@@ -28,35 +28,41 @@
                         <?php endif ?>
 
                         <!-- Side Navigation -->
```

Comment:
@ayumihamsaki your `aside` sections inside deafult.htm is breaking every page in the `Backend`.
Screenshots:
![Screenshot 2019-07-03 at 15 39 00](https://user-images.githubusercontent.com/1053320/60592218-d7acbf80-9da8-11e9-87fe-ac46cfd81249.png)
![Screenshot 2019-07-03 at 15 39 06](https://user-images.githubusercontent.com/1053320/60592219-d7acbf80-9da8-11e9-89a1-0461b5da5ab6.png)
![Uploading Screenshot 2019-07-03 at 15.39.13.png…]()

Reply:
Thanks will investigate further and try to find the cause, must be a conflict causing that.

Reply:
I might add some of this code: https://github.com/octobercms/october/pull/4293